projects/distributed-tracing/src/shared/components/table/data-cell/metric/metric-table-cell-parser.ts

```diff
@@ -0,0 +1,44 @@
+import { TableCellParser, TableCellParserBase } from '@hypertrace/components';
+import { MetricAggregation } from '../../../../graphql/model/metrics/metric-aggregation';
+import { TracingTableCellType } from '../../tracing-table-cell-type';
+
+@TableCellParser({
+  type: TracingTableCellType.Metric
+})
+export class MetricTableCellParser extends TableCellParserBase<number, number, number> {
+  public parseValue(cellData: CellData): number {
+    return Math.round(this.extractValue(cellData)!);
```

Comment:
This is pre-existing, but why are we rounding? We show decimals in places, right?

Reply:
I think this was right for Duration, but maybe not for others.